pkgs/servers/web-apps/snipe-it/update.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env nix-shell
+#! nix-shell -i bash -p nix curl jq nix-update
+
+# check if composer2nix is installed
+if ! command -v composer2nix &> /dev/null; then
+  echo "Please install composer2nix (https://github.com/svanderburg/composer2nix) to run this script."
+  exit 1
+fi
+
+CURRENT_VERSION=$(nix eval --raw '(with import ../../../.. {}; snipe-it.version)')
+TARGET_VERSION_REMOTE=$(curl https://api.github.com/repos/snipe/snipe-it/releases/latest | jq -r ".tag_name")
```

Comment:
This should optional take a GITHUB_TOKEN.

Reply:
Do we really need that? It works fine this way for me and if you run it on your local machine, you shouldn't run into any rate limits.

Reply:
I believe you that it runs fine for you but for CI systems and if many people share one IP this can become a problem.

```suggestion
TARGET_VERSION_REMOTE=$(curl ${GITHUB_TOKEN:+" -u \":$GITHUB_TOKEN\""} https://api.github.com/repos/snipe/snipe-it/releases/latest | jq -r ".tag_name")
```